great_expectations/rule_based_profiler/domain_builder/column_domain_builder.py

```diff
@@ -11,21 +11,21 @@
 class ColumnDomainBuilder(DomainBuilder):
     def __init__(
         self,
-        data_context: "DataContext",  # noqa: F821
-        batch: Optional[Batch] = None,
+        batch_list: Optional[List[Batch]] = None,
         batch_request: Optional[Union[BatchRequest, RuntimeBatchRequest, dict]] = None,
+        data_context: Optional["DataContext"] = None,  # noqa: F821
         column_names: Optional[List[str]] = None,
     ):
         """
         Args:
-            data_context: DataContext
+            batch_list: explicitly specified Batch objects foruse in DomainBuilder
```

Comment:
"for use"

Reply:
@NathanFarmer thank you!  Fixing in next commit.